examples/gnrc_border_router/Makefile.wifi.conf

```
@@ -0,0 +1,3 @@
+# Set the SSID and password of your WiFi network here
+CFLAGS += -DESP_WIFI_SSID=\"Your_WiFi_name\"
```

I would prefer to have something like the following to be able to set the parameters at make command line
```suggestion
WIFI_SSID ?= "Your_WiFi_name"
WIFI_PASS ?= "Your_secure_password"

CFLAGS += -DESP_WIFI_SSID=\"$(WIFI_SSID)\"
CFLAGS += -DESP_WIFI_PASS=\"$(WIFI_PASS)\"
```